docs/src/how-to/change-username.md

```diff
@@ -0,0 +1,19 @@
+# How to change the username
+
+If you want to change the default username to something other than `nixos`, use the `wsl.defaultUser` option.
+When building your own tarball, this should be sufficient. A user with the name specified in that option will be created automatically.
+
+Changing the username on an already installed system is possible as well.
+Follow these instructions to make sure, the change gets applied correctly:
+
+1. Change the `wsl.defaultUser` setting in your configuration to the desired username.
+2. Apply the configuration:\
+   `nixos-rebuild boot`\
+   Do not use `nixos-rebuild switch`! It may lead to the new user account being misconfigured.
```

Comment:
```suggestion
2. Apply the configuration: `nixos-rebuild boot` Do not use `nixos-rebuild switch`! It may lead to the new user account being misconfigured.
```
Is this equal to this?

Reply:
No, the backslash makes it so that there is a newline in the rendered HTML (so exactly the opposite of nearly everywhere else)